packages/core/bootstrap/src/lib/cache/index.ts

```diff
@@ -183,9 +183,10 @@ export class AdapterCache {
       feedId: adapterRequest.metricsMeta?.feedId || 'N/A',
     })
 
-    const cachedAdapterResponse = this.options.requestCoalescing.enabled
-      ? await this.getWithCoalescing(key)
-      : await this.cache.getResponse(key)
+    const cachedAdapterResponse =
+      this.options.requestCoalescing.enabled && this.options.cacheImplOptions.type === 'local'
```

Comment:
This is still disabling it for Redis

```suggestion
    const cachedAdapterResponse = this.options.requestCoalescing.enabled
```